langworld_db_data/adders/listed_value_adder.py

```diff
@@ -23,6 +23,8 @@ def add_listed_value(
         new_value_ru: str,
         custom_values_to_rename: Optional[list[str]] = None,
         index_to_assign: Union[int, None] = None,
+        description_formatted_en: Optional[str] = "",
+        description_formatted_ru: Optional[str] = "",
```

Comment:
if you use `Optional[str]`, then it's logical to make the default value `None`. Same below.
```suggestion
        description_formatted_en: Optional[str] = None,
        description_formatted_ru: Optional[str] = None,
```